src/room/InCallView.tsx

```diff
@@ -165,6 +165,13 @@ export function InCallView({
     options
   );
 
+  // TODO: move the room creation into the useRoom hook and out of the useLiveKit hook.
+  // This would than allow to not have those 4 lines
+  livekitRoom.options.audioCaptureDefaults.deviceId =
+    mediaDevices.state.get("audioinput").selectedId;
+  livekitRoom.options.videoCaptureDefaults.deviceId =
+    mediaDevices.state.get("videoinput").selectedId;
```

Comment:
NB: The options could be passed directly to the `useLiveKitRoom` below.

Reply:
yes I 100% agree. This should be refactored as mentioned in the TODO comment above